kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md

```diff
@@ -0,0 +1,125 @@
+# Oracle WebLogic Server Kubernetes Operator Tutorial #
+
+### Install and configure the operator  ###
+
+An operator is an application-specific controller that extends Kubernetes to create, configure, and manage instances of complex applications. The Oracle WebLogic Server Kubernetes Operator (the "operator") simplifies the management and operation of WebLogic domains and deployments.
+
+#### Clone the operator repository to a Cloud Shell instance ####
+First, clone the operator git repository to OCI Cloud Shell.
+```bash
+$ git clone https://github.com/oracle/weblogic-kubernetes-operator.git  -b v2.5.0
+```
+The output should be similar to the following:
+```bash
+Cloning into 'weblogic-kubernetes-operator'...
+remote: Enumerating objects: 1568, done.
+remote: Counting objects: 100% (1568/1568), done.
+remote: Compressing objects: 100% (641/641), done.
+remote: Total 123827 (delta 748), reused 1284 (delta 567), pack-reused 122259
+Receiving objects: 100% (123827/123827), 86.55 MiB | 28.25 MiB/s, done.
+Resolving deltas: 100% (74546/74546), done.
+Note: checking out 'b9d4b934ebd90df9726dc4051ece82491bd726c0'.
+
+You are in 'detached HEAD' state. You can look around, make experimental
+changes and commit them, and you can discard any commits you make in this
+state without impacting any branches by performing another checkout.
+
+If you want to create a new branch to retain commits you create, you may
+do so (now or later) by using -b with the checkout command again. Example:
+
+  git checkout -b new_branch_name
+
+Checking out files: 100% (8396/8396), done.
+```
+#### Prepare the environment ####
+In order to use Helm to install and manage the operator, you need to ensure that the service account has the `cluster-admin` role.  The default is `default` in namespace `kube-system`.  You can give that service
```

Comment:
This is only necessary for Helm 2.x.  I'm going to run through this workshop and make sure all is correct for Helm 3.